reference-implementation/lib/abstract-ops/readable-streams.js

```diff
@@ -1283,24 +1308,42 @@ function ReadableByteStreamControllerRespondInternal(controller, bytesWritten) {
 
 function ReadableByteStreamControllerRespondWithNewView(controller, view) {
   assert(controller._pendingPullIntos.length > 0);
+  assert(CanTransferArrayBuffer(view.buffer) === true);
```

Comment:
We'll change this back to `IsDetachedBuffer`. The `TransferArrayBuffer` call in this function happens before any state mutations, so it's fine.